# cv::aruco::DICT_4X4_50 is 0
dictionary_type : 0
marker_edge_size : 0.04
marker_ids : [ 0 ]